code/src/Album.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+export const Album = (props) => {
+  return (
+    <div className="album">
```

Comment:
even though we're working with React now, you could still use semantic html elements. No stress now, but keep it in mind!